gluten-ut/spark34/src/test/scala/org/apache/gluten/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -631,6 +631,7 @@ class ClickHouseTestSettings extends BackendTestSettings {
     .exclude("SPARK-37552: convert a timestamp_ntz to another time zone")
     .exclude("SPARK-38195: add a quantity of interval units to a timestamp")
     .exclude("SPARK-38284: difference between two timestamps in units")
+    .exclude("SPARK-42635: timestampadd near daylight saving transition")
```

Comment:
Why CH backend also exclude the test? What's the change for CH?

Reply:
Previously, CH only supported 3.3. This spark test was added in 3.4.